### ISSUE TITLE
Separate configuration from database model

### ISSUE THREAD

**Issue by pudo** [poderopedia/plug-and-play-1.0-RC#4]
There seems to be a lot of configuration code in the database model. Splitting these up might make for a clearer separation of concerns. 

https://github.com/poderopedia/plug-and-play-1.0-RC/blob/master/poderopedia/models/db.py#L112

**Comment by juaneduardo** [comment]
I will update soon... Thanks for the report.

**Comment by miguelpaz** [comment]
Pudo: Can you provide some specific examples and some sugestions on splitting these up?

**Comment by miguelpaz** [comment]
Working on it now. Need to test with interested users @pudo ?

**Comment by pudo** [comment]
Yep :) Was hoping to sit down this weekend and set up an instance. Do you
guys have an IRC room or mailing list?
